lib/engine/game/g_1880/share_pool.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_relative '../../share_pool'
+
+module Engine
+  module Game
+    module G1880
+      class SharePool < Engine::SharePool
+        def log_sell_shares(entity, verb, bundle, price, swap_text)
+          fee = additional_price_adjustments(bundle)
+          @log << "#{entity.name} #{verb} #{num_presentation(bundle)} " \
+                  "of #{bundle.corporation.name} and receives #{@game.format_currency(price)}" \
+                  " (broker fee: #{@game.format_currency(fee)})#{swap_text}"
+        end
+
+        def additional_price_adjustments(bundle)
```

Comment:
Maybe call this method broker_fee?

Reply:
i'm overwriting a hook in the base code that is used in sell_shares, if i rename it broker_fee here, i'll need to do it there, additional price adjustment is more generic